### ISSUE TITLE
Reconsider directly editing the card title

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/deck#5310]
As reported in the design chat:

> The fact that clicking on a card has two distinct actions even though it looks like it only has one. I'm talking about the action to open a card when you click on it and the action to edit a card title inline when you click on it's title.
To me, the functionality to edit a card title should be removed to make the card behave like it looks and also make it an easier click target.

My reply:
> Iirc Trello used to have a similar behavior that’s why it has been like that since the initial implementation but these days Trello also opens the card

I think opening the card is the most common use case after creating, so maybe we can get rid of the click title to edit the title inline function and instead move always open the card details view where you can edit the title as well.

As a middle ground we could also add a rename/change title option to the menu and let this still trigger the inline editing without opening the modal/sidebar but that means another menu entry which adds up to the length of the menu.  Might still be useful for a rename keyboard shortcut with #3758 

@nextcloud/designers Any thoughts?

**Comment by pschopen** [comment]
Absolutely agree. Makes it also easier to move cards and not accidentally edit the title.

**Comment by marcoambrosini** [comment]
I also agree that we should remove direct title editing from the card

**Comment by nousernameavailableanymore** [comment]
I agree. Please remove direct editing of card title. This is very confusing and janky.

**Comment by jancborchardt** [comment]
@juliushaertl yup – if in doubt, let’s do it like Trello. :)

**Comment by narFnarF** [comment]
At the moment, there seems to be a bug that prevent editing the title of a card from the popup. See #6250
